app/views/no_ta_enrolled_alert_mailer/email.html.haml

```diff
@@ -0,0 +1,34 @@
+%link{rel: 'stylesheet', href:'/mailer.css'}
+
+%table.row
+  %tbody
+    %tr
+      %th
+        %table
+          %tr
+            %td.main-content
+              %p.paragraph
+                = "Hello #{@greeted_users},"
```

Comment:
Hi [users].

You indicated that your course has an additional instructor or TA support. It looks like you haven't added anyone else in the instructor role for your dashboard.wikiedu.org course page yet.

Please follow these steps to add an additional instructor:

* Navigate to your [course page](link to course page].
* Click 'Edit Details'.
* Click the '+' icon next to your name in the 'Instructors'.
* Enter the name, username and role (eg, "TA") and click 'Enroll'.

Feel free to get in touch with us if you have any questions.

Best regards,
[name]

Reply:
![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/92005669/bcdd8627-716c-4e6a-856c-24924e82e35d)